runtime/server/chat.go

```diff
@@ -129,10 +130,18 @@ func (s *Server) Complete(ctx context.Context, req *runtimev1.CompleteRequest) (
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	var chatCtx ai.MessageContext
+	chatCtx.Explore = req.Explore
+	chatCtx.Dimensions = req.Dimensions
+	chatCtx.Measures = req.Measures
+	chatCtx.Where = metricsview.NewExpressionFromProto(req.Where)
+	chatCtx.TimeRange = req.TimeRange
```

Comment:
See above suggestion about using `AnalystAgentArgs` for this. But also, it's less verbose to do it in when initializing the struct:
```suggestion
	args := &ai.AnalystAgentArgs{
		Explore: req.Explore,
		Dimensions: req.Dimensions,
		Measures: req.Measures,
		Where: metricsview.NewExpressionFromProto(req.Where),
		TimeRange: req.TimeRange,
	}
```